Description: Remove Test_spell_single_word()
 Test_spell_single_word() test was failing after CVE-2022-1621 patch
Author: Nishit Majithia <nishit.majithia@canonical.com>
Origin: other
--- vim-8.1.2269.orig/src/testdir/test_spell.vim
+++ vim-8.1.2269/src/testdir/test_spell.vim
@@ -485,16 +485,6 @@ func Test_spell_screendump()
   call delete('XtestSpell')
 endfunc
 
-func Test_spell_single_word()
-  new
-  set spell
-  silent! norm 0R00
-  spell! ßÂ
-  silent 0norm 0r$ Dvz=
-  set nospell
-  bwipe!
-endfunc
-
 let g:test_data_aff1 = [
       \"SET ISO8859-1",
       \"TRY esianrtolcdugmphbyfvkwjkqxz-\xEB\xE9\xE8\xEA\xEF\xEE\xE4\xE0\xE2\xF6\xFC\xFB'ESIANRTOLCDUGMPHBYFVKWJKQXZ",
